homeassistant/components/myuplink/strings.json

```diff
@@ -41,5 +41,25 @@
         "name": "Status"
       }
     }
+  },
+  "exceptions": {
+    "config_entry_auth_failed": {
+      "message": "Error while logging in to the API."
+    },
+    "config_entry_not_ready": {
+      "message": "Error while loading the integration."
+    },
+    "incorrect_oauth2_scope": {
+      "message": "Stored permissions are deprecated and are refreshed automatically."
```

Comment:
Hmm, I would not call it deprecated, maybe just out of date (as in, you might have this in the future again, so let's pick a message that works)
And, we don't refresh automatically, we require manual intervention from the user right?